ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
@lindong28 
There is actually a use case for sudo per ssh_command - we do have some system tests which require sudo as root for a few individual commands.